src/nyc_trees/apps/core/mail/backends/boto.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+
+import boto
+import json
+import logging
+import traceback
+
+from django.conf import settings
+from django.core.mail.backends.base import BaseEmailBackend
+
+from django_statsd.clients import statsd
+
+logger = logging.getLogger(__name__)
+
+
+class EmailBackend(BaseEmailBackend):
+
+    def __init__(self, fail_silently=False, aws_access_key_id=None,
+                 aws_secret_access_key=None, aws_region=None):
+        super(EmailBackend, self).__init__(fail_silently=fail_silently)
+        self.aws_access_key_id =\
+            aws_access_key_id or settings.EMAIL_AWS_ACCESS_KEY_ID
+        self.aws_secret_access_key =\
+            aws_secret_access_key or settings.EMAIL_AWS_SECRET_ACCESS_KEY
+        self.aws_region =\
+            aws_region or settings.EMAIL_AWS_REGION
+
+    def open(self):
+        try:
+            if not self.connection:
+                self.connection = boto.ses.connect_to_region(
+                    self.aws_region,
+                    aws_access_key_id=self.aws_access_key_id,
+                    aws_secret_access_key=self.aws_secret_access_key)
+                self._log_connect()
+        except:
+            self._log_connect_failure(traceback.format_exc())
+            if not self.fail_silently:
+                raise
+
+    def close(self):
+        # SES connections are not stateful, so there is no cleanup needed
+        self.connection = None
+
+    def send_messages(self, email_messages):
+        if not email_messages:
+            return
+        sent_message_count = 0
+        if not self.open():
```

Comment:
I don't see a `return` statement in `self.open()`.  Where does this truthy return value come from when we connect succefully?